sdcm/utils/docker_remote.py

```diff
@@ -41,6 +42,16 @@ def private_ip_address(self):
     def cql_ip_address(self):
         return self.internal_ip_address
 
+    @cached_property
+    def running_in_docker(self):
+        ok = self.node.remoter.run("test /.dockerenv", ignore_status=True).ok
+        ok |= 'docker' in self.node.remoter.run('ls /proc/self/cgroup', ignore_status=True).stdout
+        return ok
+
+    @property
```

Comment:
If "running_in_docker" is cached then this one won't have changes after first calculation.
So, need to make this one be cached too.

Reply:
one function runs commands via ssh, hence caching it.
one just do an if statement, caching less critical.

So my POV it's nice to have, not *needed*

Reply:
changed